### PR TITLE
rpc: `masternode winners` results should also include operator addresses

### PR DESCRIPTION
**Pull request by UdjinM6**
~Based on #3861 atm~

### CODE REVIEW

**Comment by PastaPastaPasta** [comment]
Looks good overall, wanna write a test?

**Comment by thephez** [comment]
> Looks good overall, wanna write a test?

:cricket: :cricket: :cricket: :smile:

**Comment by UdjinM6** [comment]
Done. Had to rebase to include `rpcmasternode.py` from recent changes.

File: test/functional/test_framework/test_framework.py [hunk 1]
```diff
@@ -506,7 +506,7 @@ def _initialize_chain_clean(self):
 
 
 class MasternodeInfo:
-    def __init__(self, proTxHash, ownerAddr, votingAddr, pubKeyOperator, keyOperator, collateral_address, collateral_txid, collateral_vout):
+    def __init__(self, proTxHash, ownerAddr, votingAddr, pubKeyOperator, keyOperator, ipAndPort, operatorReward, operatorPayoutAddress, collateral_address, collateral_txid, collateral_vout):
```
Comment (xdustinface) [inline test/functional/test_framework/test_framework.py:509]:
```suggestion
    def __init__(self, proTxHash, ownerAddr, votingAddr, pubKeyOperator, keyOperator, collateral_address, collateral_txid, collateral_vout):
```

File: test/functional/test_framework/test_framework.py [hunk 2]
```diff
@@ -603,7 +607,10 @@ def prepare_masternode(self, idx):
 
         self.nodes[0].generate(1)
 
-        self.mninfo.append(MasternodeInfo(proTxHash, ownerAddr, votingAddr, bls['public'], bls['secret'], address, txid, collateral_vout))
+        if operatorReward > 0:
+            self.nodes[0].protx('update_service', proTxHash, ipAndPort, bls['secret'], operatorPayoutAddress, address)
+
+        self.mninfo.append(MasternodeInfo(proTxHash, ownerAddr, votingAddr, bls['public'], bls['secret'], ipAndPort, operatorReward, operatorPayoutAddress, address, txid, collateral_vout))
```
Comment (xdustinface) [inline test/functional/test_framework/test_framework.py:613]:
```suggestion
        self.mninfo.append(MasternodeInfo(proTxHash, ownerAddr, votingAddr, bls['public'], bls['secret'], address, txid, collateral_vout))
```

**Review by xdustinface — CHANGES_REQUESTED** [review]
The changes in `MasternodeInfo` look like leftover and should probably be dropped or used somehow but currently they are not even assigned in `__init__`.

**Review by xdustinface — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Pull request merged**